app/models/paper.rb

```diff
@@ -1,10 +1,31 @@
 class Paper < ApplicationRecord
   acts_as_taggable
+  acts_as_taggable_on :biases, :methodologies
+  ActsAsTaggableOn.strict_case_match = true
 
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references, dependent: :destroy
 
   accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
   validates_associated :authors
+  validate :allowed_biases, :allowed_methodologies
+
+  def allowed_biases
+	  invalid_biases = bias_list - valid_biases
+	  invalid_biases.each {|b| errors.add(b, 'is not an accepted bias') }
+	end
+
+  def valid_biases
+    %w(selection performance detection attrition reporting small\ sample\ size)
```

Comment:
To make it easier to edit these in the future, it might be simpler to break these into separate lines and just use the normal array syntax:

```ruby
[
  'selection',
  'performance',
  'detection',
  'attrition',
  'reporting',
  'small sample size'
]
```